man/man8/zpool.8

```diff
@@ -1571,6 +1571,15 @@ Scripted mode. Do not display headers, and separate fields by a single tab inste
 .sp
 .ne 2
 .na
+\fB\fB-k\fR\fR
+.ad
+.RS 12n
+Display the kind of device a pool or vdev is based on: ssd or hdd.  Mixed mirrored devices that have at least one solid-state member are marked "mix".  A pool is considered mixed if all members are at least mixed.
```

Comment:
Thanks!  Added `A file vdev is assumed to have ssd characteristics when evaluating parents.`

Also changed the entries in the kind column to be right-aligned.